dev/postgres-docker/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.5"
+services:
+  postgres:
+    build:
+      context: .
+      dockerfile: postgresql.dockerfile
+    ports:
+      - 5432:5432/tcp
+    environment:
+      POSTGRES_PASSWORD: sormas
+    restart: unless-stopped
+    volumes:
+      - ./psql.conf:/etc/postgresql/postgresql.conf
+    command: postgres -c config_file=/etc/postgresql/postgresql.conf
```

Comment:
Maybe setting `max_prepared_transactions` is sufficient in a dev environment (and this avoids maintaining psql.conf)

  `command: -c 'max_prepared_transactions=64'`

Reply:
Resolved by the latest commit, since that uses the pre-configured [hzibraunschweig/sormas-postgres](https://hub.docker.com/r/hzibraunschweig/sormas-postgres) Docker image.